components/ws-proxy/pkg/sshproxy/server.go

```diff
@@ -284,6 +285,9 @@ func (s *Server) HandleConn(c net.Conn) {
 	clientConn, clientChans, clientReqs, err := ssh.NewServerConn(c, s.sshConfig)
 	if err != nil {
 		c.Close()
+		if errors.Is(err, io.EOF) {
+			return
```

Comment:
Should first invoke `ReportSSHAttemptMetrics(err)`, and then do this block?

Reply:
Hmm, can EOF happen for any valid use cases? Like, if I disconnect network cable or turn off wifi?

Reply:
Gotcha. So, after this change, we'll probably expect error_type OTEHRS for with failed status to come down (a lot of it is probably EOF).

![image](https://github.com/gitpod-io/gitpod/assets/1272076/ece41cd5-6ce7-44d1-9a8e-9dc4b62a82f4)